src/event/event_emitter.cairo

```diff
@@ -337,6 +387,55 @@ mod EventEmitter {
         reason_bytes: Array<felt252>
     }
 
+    #[derive(Drop, starknet::Event)]
+    struct OrderCreated {
+        key: felt252,
+        order: Order
```

Comment:
Do you need the whole struct of Order implemeneted in OrderCreated ?

Reply:
I checked from here https://github.com/gmx-io/gmx-synthetics/blob/77a2ff39f1414a105e8589d622bdb09ac3dd97d8/contracts/order/OrderEventUtils.sol#L21

Is it wrong?